### PR TITLE
Update FontAwesome to 4.7.0

### PR DESCRIPTION
**Pull request by drzax**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/font-fontawesome.rb [hunk 1]
```diff
@@ -1,12 +1,13 @@
 cask 'font-fontawesome' do
-  version '4.6.3'
-  sha256 'e59dd2ee7f461a15fbc08ea8156d807b6d1401e2274701cf093ef2e5b47b0ba0'
+  version '4.7.0'
+  sha256 '35d79ae86c8d4fd0f931843ec969191700940875bb941044a4c860eb200f166c'
 
-  url "https://fortawesome.github.io/Font-Awesome/assets/font-awesome-#{version}.zip"
+ # github.com/FortAwesome was verified as official by following link in footer of homepage
```
Comment (vitorgalvao) [inline Casks/font-fontawesome.rb]: No. This needs to follow our template (i.e. “was verified as official when first introduced to the cask”).

Comment (vitorgalvao) [inline Casks/font-fontawesome.rb]: Also, please make it a bit more specific (`github.com/FortAwesome/Font-Awesome`).

**Comment by vitorgalvao** [comment]
Was able to make the changes myself. Just waiting for Travis.

**Comment by drzax** [comment]
That wording makes no sense to me so I made it make sense.

Feel free to change it, I can't spend any more time on this today.


On 25 October 2016 at 9:49:04 am, Vítor Galvão (notifications@github.com)
wrote:

*@vitorgalvao* requested changes on this pull request.
------------------------------

In Casks/font-fontawesome.rb
<https://github.com/caskroom/homebrew-fonts/pull/900#pullrequestreview-5561430>
:

>
-  url "https://fortawesome.github.io/Font-Awesome/assets/font-awesome-#{version}.zip"
+ # github.com/FortAwesome was verified as official by following link
in footer of homepage

No. This needs to follow our template (i.e. “was verified as official when
first introduced to the cask”).

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/caskroom/homebrew-fonts/pull/900#pullrequestreview-5561430>,
or mute the thread

**Pull request merged**